VerbalExpressionsUnitTests/SanatizeTests.cs

```diff
@@ -8,7 +8,6 @@ namespace VerbalExpressionsUnitTests
     public class SanatizeTests
```

Comment:
To quote Homer Simpson, "it was like that when I got here".

aka "outside the scope of this pull request".